app/controllers/users_controller.rb

```diff
@@ -12,9 +18,26 @@ def create
     end
   end
 
+  def update_role
+    @user = User.find(params[:id])
+    
+    if @user.user?
+      @user.update(role: 'admin')
+      puts @user.errors.full_messages
```

Comment:
We never print logs like this. You can study about [rails logger](https://guides.rubyonrails.org/debugging_rails_applications.html#the-logger) and use that if you wish.

Reply:
ok thank you very much for providing this for me, i will take a look at it and probably delete this spec